ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host2/node_type.yml

```diff
@@ -10,7 +8,7 @@ pe:
     bgp_as: 42
     id: 42
   nodes:
-    host2:
+    - name: host2
```

Comment:
Can we remove this file?

Reply:
I think I prefer to keep it for now, since we will have new keys to deprecate soon.

Reply:
okay.